src/assets/sass/protocol/components/_feature-card.scss

```diff
@@ -0,0 +1,214 @@
+    }
+
+    .mzp-c-card-feature-title {
+        @include text-display-sm;
+    }
+
+    .mzp-c-card-feature-desc {
+        @include text-body-md;
+        color: $color-gray-70;
+        margin: $margin-xs 0;
+    }
+
+    .mzp-c-card-feature-list li {
+        margin-bottom: $margin-xl;
+
+        &:last-child {
+            margin-bottom: 0;
+        }
+    }
+
+    // dark theme color styling
+    .mzp-t-dark & {
+        color: $color-white;
+
+        .mzp-c-card-feature-desc {
+            color: $color-gray-30;
+        }
+    }
+
+    @media #{$mq-md} {
+        margin-bottom: $layout-2xl;
+        max-width: 800px;
+
+        .mzp-c-card-feature-list li {
+            margin-bottom: $margin-2xl;
+        }
+
+        &.mzp-l-card-feature-left-half,
+        &.mzp-l-card-feature-right-half,
+        &.mzp-l-card-feature-left-third,
+        &.mzp-l-card-feature-right-third {
+            @include clearfix;
+            margin-bottom: $layout-xl;
+            max-width: none;
+            position: relative;
+
+            .mzp-c-card-feature-content,
+            .mzp-c-card-feature-media-wrapper {
+                margin-top: 0;
+            }
+        }
+
+        &.mzp-l-card-feature-left-half,
+        &.mzp-l-card-feature-right-half {
+            .mzp-c-card-feature-content,
+            .mzp-c-card-feature-media-wrapper {
+                width: calc(50% - (#{$margin-xl} / 2));
+            }
+        }
+
+        &.mzp-l-card-feature-left-third,
+        &.mzp-l-card-feature-right-third {
+            .mzp-c-card-feature-content {
+                width: calc(41% - #{$margin-2xl});
+            }
+
+            .mzp-c-card-feature-media-wrapper {
+                width: 59%;
+            }
+        }
+
+        &.mzp-l-card-feature-left-half,
+        &.mzp-l-card-feature-left-third {
+            .mzp-c-card-feature-content {
+                @include bidi((
+                    (float, left, right),
+                    (left, 0, right, auto),
+                ));
+            }
+
+            .mzp-c-card-feature-media-wrapper {
+                @include bidi((
+                    (float, right, left),
+                ));
+            }
+        }
+
+        &.mzp-l-card-feature-right-half,
+        &.mzp-l-card-feature-right-third {
+            .mzp-c-card-feature-content {
+                @include bidi((
+                    (float, right, left),
+                    (right, 0, left, auto),
+                ));
+            }
+
+            .mzp-c-card-feature-media-wrapper {
+                @include bidi((
+                    (float, left, right),
+                ));
+            }
+        }
+    }
+
+    @supports(display: grid) {
+        @media #{$mq-md} {
+            &.mzp-l-card-feature-left-half,
+            &.mzp-l-card-feature-right-half,
+            &.mzp-l-card-feature-left-third,
+            &.mzp-l-card-feature-right-third {
+                @include grid-column-gap($margin-xl);
```

Comment:
Nit: Since this isn't technically a margin you can use the generic `$spacing-2xl` token. I'm still not sure we need separate tokens for margins and padding since they all have the same values and are just aliased to the spacing tokens; perhaps that's more bikeshedding we should do soon.